templates/base.html

```diff
@@ -5,12 +5,13 @@
     <meta charset="UTF-8">
     <meta name="apple-mobile-web-app-capable" content="yes">
     <meta name="viewport" content="width=device-width">
-        <!-- TWITTER -->
-        <meta name="twitter:card" content="summary_large_image">
-        <meta name="twitter:site" content="@urbit">
-        <meta name="twitter:creator" content="@urbit">
-        <!-- END TWITTER -->
-    <meta name="description" content="{% block description %}{% endblock description %}">
+    <meta name="twitter:card" content="summary_large_image">
```

Comment:
What image is used here?

Reply:
![image](https://user-images.githubusercontent.com/20846414/67230323-f4214600-f3f1-11e9-9f6b-29eb2d0fdde7.png)